### PR TITLE
Edit mode interaction prevented

### PR DESCRIPTION
**Pull request by ahmed-bazzara**

### CODE REVIEW

**Comment by ahmed-bazzara** [comment]
JIRA ticket

https://tretton37.atlassian.net/secure/RapidBoard.jspa?rapidView=33&projectKey=QPE&modal=detail&selectedIssue=QPE-412

**Review by kristofferlind — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -54,13 +54,10 @@ export default {
       var self = this;
       self.$scope.g.self = self; // Save reference for call to backendApi
 
-      // Only repaint here when in edit mode
       self.$scope.g.editMode = (self.options.interactionState == 2);
-      if (self.$scope.g.editMode) {
```
Comment (kristofferlind) [inline src/index.js]: should possibly leave this in there, an ng directive is loaded that then has a couple of watches for scope changes, so unless you've seen an issue that is fixed by removing this i think it's more performant to leave it in there.

**Review by kristofferlind — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (ahmed-bazzara) [reply src/index.js]: Hmmm, this causes the extension to only re-paint on edit mode!

Comment (kristofferlind) [reply src/index.js]: If you know there's a scenario where it doesn't repaint when it should in analysis mode i'm fine with it, but there are watches for data changes, chart resize and selection. Which is why I think this is only needed for editmode, where there are also settings changes.

https://github.com/qlik-oss/barsPlus/blob/master/src/barsPlus-directive.js#L133-L184

Comment (ahmed-bazzara) [reply src/index.js]: reverting it back and it actually does re-paints ,, but also that variable `editMode` is always false even on edit-mode

Comment (ahmed-bazzara) [reply src/index.js]: @kristofferlind so, should we leave it there,, although it's never true?

Comment (kristofferlind) [reply src/index.js]: in that case it's just dead code and should be cleaned up

**Review by kristofferlind — APPROVED** [review]

**Pull request merged**